src/runtime/threadpool/tests.rs

```diff
@@ -113,3 +113,18 @@ fn block_on_std_01_spawn() {
     // is that we're able to spawn it successfully.
     rt.block_on_std(async { tokio_01::spawn(futures_01::future::lazy(|| Ok(()))) });
 }
+
+#[test]
+fn tokio_02_blocking_shutdown() {
+    let ran = Arc::new(AtomicBool::new(false));
+    let ran2 = ran.clone();
+    super::run_std(async move {
+        println!("in future, before blocking");
```

Comment:
probably don't need these prints?

Reply:
i found them helpful, and it's a test. can remove them if you prefer.

Reply:
nah lets merge this :P